public/app/plugins/datasource/jaeger/datasource.ts

```diff
@@ -77,11 +80,7 @@ export class JaegerDatasource extends DataSourceWithBackend<JaegerQuery, JaegerJ
       return of({ data: [emptyTraceDataFrame] });
     }
 
-    if (
-      config.featureToggles.jaegerBackendMigration &&
-      // No query type means that the query is a trace ID query
-      (!target.queryType || target.queryType === 'dependencyGraph')
-    ) {
+    if (config.featureToggles.jaegerBackendMigration) {
```

Comment:
```suggestion
    if (config.featureToggles.jaegerBackendMigration && target.queryType !== 'upload') {
```
Upload query will throw error